tekton/utils.go

```diff
@@ -161,9 +161,9 @@ func GetOutputImageDigest(object client.Object) (string, error) {
 // GetComponentSourceGitUrl returns a string containing the CHAINS-GIT_URL result value from a given PipelineRun.
 func GetComponentSourceGitUrl(object client.Object) (string, error) {
 	if pipelineRun, ok := object.(*tektonv1.PipelineRun); ok {
-		for _, pipelineResult := range pipelineRun.Status.Results {
-			if pipelineResult.Name == "CHAINS-GIT_URL" {
-				return pipelineResult.Value.StringVal, nil
+		for _, param := range pipelineRun.Spec.Params {
```

Comment:
I'm not sure that we want to do make this change for all types of build pipelineRuns.
If we don't, we should have a separate function that fetches the source git URL and is only used when the build pipelineRuns is running for a `pull_request` event.

Reply:
IMO it's safe to use git-url in push events, as it's supposed to be source url. In case of push events source and target URL in PAC are the same value.

Reply:
Okay, in that case I think we should be fine as long as we document that the integration service now depends on thhe git-url parameter being present on the build pipelineRun. I think at least [this section](https://redhat-appstudio.github.io/book/book/integration-service.html#image-extraction-details) in the book repo should be updated.

Reply:
Can do.  I'm also going to update [this function](https://github.com/redhat-appstudio/integration-service/blob/main/tekton/utils.go#L174) to pull from `Spec.Params`.  I'm bringing up another cluster to make there's a param called `revision` or `commit` or something similar.  I think it's best to get both the source git url and commit from a similar place. It'll make the code easier to understand.

Reply:
[Book PR](https://github.com/redhat-appstudio/book/pull/151)

Reply:
It is now, thanks for the catch. My last push failed.